checks/evaluation/sast_test.go

```diff
@@ -79,6 +87,7 @@ func TestSAST(t *testing.T) {
 			result: scut.TestReturn{
 				Score:        10,
 				NumberOfInfo: 3,
+				NumberOfWarn: 1,
```

Comment:
I adjusted such that https://github.com/ossf/scorecard/blob/eefb6bfe4e50ca6902f3a5c3d7bf38afa6b4c674/checks/evaluation/sast.go#L159-L161 doesn't warn simply in case if CodeQL isn't there, and similarly with Snyk. If it need further refinements it may be smarter to create an issue and follow-up